worker/src/main/java/gov/cms/ab2d/worker/processor/PatientClaimsProcessorImpl.java

```diff
@@ -154,90 +76,59 @@ private void handleException(StreamHelper helper, String data, Exception e) thro
+        OffsetDateTime startDate = getStartDate(contract);
+        if (attestDate.isAfter(startDate)) {
+           return attestDate;
+        }
+        return startDate;
+    }
 
+    private OffsetDateTime getStartDate(String contract) {
         String dateToUse = startDate;
         if (isContractSpecial(contract)) {
             dateToUse = startDateSpecialContracts;
         }
-        Date date;
-        try {
-            date = sdf.parse(dateToUse);
-        } catch (ParseException e) {
-            LocalDateTime d = LocalDateTime.of(2020, 1, 1, 0, 0, 0, 0);
-            date = new Date(d.toInstant(ZoneOffset.UTC).toEpochMilli());
-        }
+        OffsetDateTime date = OffsetDateTime.parse(dateToUse + " 00:00:00:000+00:00",
+                DateTimeFormatter.ofPattern("dd/MM/uuuu HH:mm:ss:SSSXXXXX"));
+
         return date;
     }
 
     private boolean isContractSpecial(String contract) {
         return this.specialContracts != null && !this.specialContracts.isEmpty() && specialContracts.contains(contract);
     }
 
-    List<Resource> extractResources(String contractNum, List<BundleEntryComponent> entries,
-                                    final List<FilterOutByDate.DateRange> dateRanges,
-                                    OffsetDateTime attTime, Map<String, ContractBeneficiaries.PatientDTO> patientsMap) {
-        if (attTime == null) {
-            return new ArrayList<>();
-        }
-        long epochMilli = attTime.toInstant().toEpochMilli();
-        Date attDate = new Date(epochMilli);
-        final Date earliestDate = getStartDate(contractNum);
+    List<Resource> extractResources(List<Bundle.BundleEntryComponent> entries) {
         return entries.stream()
                 // Get the resource
-                .map(BundleEntryComponent::getResource)
+                .map(Bundle.BundleEntryComponent::getResource)
                 // Get only the explanation of benefits
                 .filter(resource -> resource.getResourceType() == ResourceType.ExplanationOfBenefit)
                 // Filter by date
-                .filter(resource -> skipBillablePeriodCheck || FilterOutByDate.valid((ExplanationOfBenefit) resource, attDate, earliestDate, dateRanges))
+                // .filter(resource -> skipBillablePeriodCheck || FilterOutByDate.valid((ExplanationOfBenefit) resource, attDate, earliestDate, dateRanges))
                 // filter it
                 .map(resource -> ExplanationOfBenefitTrimmer.getBenefit((ExplanationOfBenefit) resource))
                 // Remove any empty values
                 .filter(Objects::nonNull)
                 // Remove Plan D
                 .filter(resource -> !isPartD(resource))
                 // Make sure the returned patient ID is actually part of the contract
-                .filter(resource -> validPatientInContract(resource, patientsMap))
+                //.filter(resource -> validPatientInContract(resource, patientsMap))
```

Comment:
Do these lines still need to be commented out?

Reply:
Yes. I left them in here for the time being because I want it obvious it has moved out of this section. We don't have the full list of patients and when they were members of the contract at this point.